docs/builds/guides/frameworks/angular.md

```diff
@@ -299,6 +331,171 @@ Then, add it in the `angular.json` configuration file.
 }
 ```
 
+## Accessing the CKEditor API from an Angular Component
```

Comment:
This part misses the *normal* way to access it (https://github.com/ckeditor/ckeditor5-angular/issues/61#issuecomment-464508361).

```html
<ckeditor (ready)="onReady($event)">
```

```js
@Component()
class MyComponent {
  onReady( editor ){
    // doSomethingWithEditor( editor );    
  }
}
```